naga/src/front/spv/mod.rs

```diff
@@ -4876,6 +4876,11 @@ impl<I: Iterator<Item = u32>> Frontend<I> {
                 let low = self.next()?;
                 match width {
                     4 => crate::Literal::U32(low),
+                    8 => {
+                        inst.expect(5)?;
```

Comment:
Why 5 here? What does this do?

Reply:
Makes sure there's at least 4 bytes more to read for the second half of the u64 literal, and that we're not at the file end, if i understand correctly. Its a common pattern in this file